node/resolvers/Mutations/Users.ts

```diff
@@ -11,6 +11,43 @@ import {
 
 const config: any = currentSchema('b2b_users')
 
+const MAX_RETRY = 5
+
+const delay = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms))
+const setChangeSession = async (
+  sessionParameters: any,
+  countRetry = 0
+): Promise<void> => {
+  const {
+    context: {
+      clients: { session },
+      vtex: { logger },
+    },
+    publicKey,
+    value,
+    sessionCookie,
+  } = sessionParameters
+
+  try {
+    await session.updateSession(publicKey, value, [], sessionCookie)
+  } catch (error) {
+    logger.error({
+      error,
+      message: 'setChangeSession.error',
+      attempt: countRetry,
+    })
+
+    if (countRetry < MAX_RETRY) {
+      countRetry++
+      const backoff = 2 ** countRetry * 500
```

Comment:
can we define a constant for the `500`? Maybe `RETRY_BACKOFF_FACTOR_MS` ?

also, should the initial backoff be a little lower maybe? I wonder if instead of 500 for the factor we could use 200? So first backoff will be fast (400ms) and last one long (2s)

Reply:
Changed to be as long as 1 second since the backoff cumulative result will be 2500ms